soe/php7.0/Dockerfile

```diff
@@ -0,0 +1,76 @@
+# Standard Operating Environment running off PHP 7.0.
+
+FROM chinthakagodawita/php:7.0
+MAINTAINER Chinthaka Godawita <chin.godawita@me.com>
+
+# Install all packages.
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install socat ssmtp
+
+# Compile xdebug from source.
```

Comment:
Can we try using the version of XDebug from the repository rather than compiling this?

Package `xdebug`: https://launchpad.net/~ondrej/+archive/ubuntu/php-7.0